src/core/src/main/java/io/github/lonamiwebs/stringlate/classes/git/GitWrapper.java

```diff
@@ -218,11 +218,10 @@ public static File findProperIcon(final RepositoryResources resources, final int
             i--;
         }
         // Just return a random one, this shouldn't happen
-        if (i >= 0 && i < foundIcons.size()) {
-            return foundIcons.get(i);
-        } else {
-            return null;
+        if (i >= 0) {
+            foundIcons.size();
```

Comment:
This is a no-op, which I'm not sure what the purpose is. I think you should revert this part.